File: CourceObjective-C/SegmentedExample/SegmentedExample/ViewController.m
//
//  ViewController.m
//  SegmentedExample
//
//  Created by Vanilson NiLL on 04/07/2018.
//  Copyright © 2018 Vanilson NiLL. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()
- (IBAction)OnSegmentedChange:(id)sender;

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


- (IBAction)OnSegmentedChange:(id)sender {
    UISegmentedControl *seg = (UISegmentedControl *) sender;
    if([seg selectedSegmentIndex] == 0) {
        _labelSunday.hidden = NO;
        _labelMonday.hidden = YES;
        _labelTuesday.hidden = YES;
        _labelWednesday.hidden = YES;
    }
    if([seg selectedSegmentIndex] == 1) {
        _labelSunday.hidden = YES;
        _labelMonday.hidden = NO;
        _labelTuesday.hidden = YES;
        _labelWednesday.hidden = YES;
    }
    if([seg selectedSegmentIndex] == 2) {
        _labelSunday.hidden = YES;
        _labelMonday.hidden = YES;
        _labelTuesday.hidden = NO;
        _labelWednesday.hidden = YES;
    }
    if([seg selectedSegmentIndex] == 3) {
        _labelSunday.hidden = YES;
        _labelMonday.hidden = YES;
        _labelTuesday.hidden = YES;
        _labelWednesday.hidden = NO;
    }
}
@end

File: App01CourseIOS/App01CourseIOS/ViewController.swift
//
//  ViewController.swift
//  App01CourseIOS
//
//  Created by Vanilson on 04/04/17.
//  Copyright © 2017 Vanilsonvl. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    @IBOutlet weak var ageField: UITextField!
    @IBOutlet weak var resultText: UILabel!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    @IBAction func getAge(_ sender: Any) {
        resultText.text = "A idade do cachorro é " + String(Int(ageField.text!)! * 7)
    }
    
    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

